redash/query_runner/pg.py

```diff
@@ -23,13 +24,6 @@
 
 logger = logging.getLogger(__name__)
 
-try:
-    import boto3
-
-    IAM_ENABLED = True
-except ImportError:
-    IAM_ENABLED = False
-
```

Comment:
I moved boto3 to the main group in the pull request below, so I removed the check.

- https://github.com/getredash/redash/pull/7302/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711